### PR TITLE
Bug fixing: bigmler reify failed for dataset generated from batch res…

### PR DESCRIPTION
**Pull request by mmerce**
…ources

### CODE REVIEW

File: bigmler/prediction.py [hunk 1]
```diff
@@ -467,6 +467,7 @@ def draw_progress_bar(current, total):
         """
         pct = 100 - ((total - current) * 100) / (total)
         console_log("Predicted on %s out of %s models [%s%%]" % (
+            #localize(current), localize(total), pct), reset=True)
```
Comment (jaor) [inline bigmler/prediction.py]: leftover?

File: bigmler/reify/restchain.py [hunk 2]
```diff
@@ -223,20 +230,28 @@ def reify_dataset(self, resource_id):
...
         if origin in ['origin_batch_resource', 'cluster']:
             if origin == "cluster":
                 opts['create'].update({"centroid": child['centroid']})
-            _, grandparent = u.get_origin_info(parent)
+            grandparents = u.get_origin_info(parent)
+            if origin == "origin_batch_resource" and \
+                    isinstance(grandparents, list):
+                for gp_origin, grandparent in grandparents:
+                    if gp_origin == "dataset":
+                        break
```
Comment (jaor) [inline bigmler/reify/restchain.py]: hmm.... this for loop does nothing, no? it just stops looping when `gp_origin` is dataset, but does nothing when it isn't... did you accidentally remove other code?

File: bigmler/reify/restutils.py [hunk 3]
```diff
@@ -200,11 +200,11 @@ def common_dataset_opts(resource, referrer, opts, call="create"):
     fields_attributes = get_fields_changes(resource, referrer=referrer)
     if fields_attributes:
         opts[call].update({"fields": fields_attributes})
-
-    # input fields
-    input_fields = get_input_fields(resource, referrer=referrer)
-    if input_fields:
-        opts[call].update({'input_fields': input_fields})
+    if call=="create":
```
Comment (jaor) [inline bigmler/reify/restutils.py]: nit: spaces around `==`

File: bigmler/prediction.py (hunk 1 above)
Comment (mmerce) [inline bigmler/prediction.py]: intentionally. Next bindings version need this change and this version came in between. Just to remind me.

File: bigmler/reify/restchain.py (hunk 2 above)
Comment (mmerce) [inline bigmler/reify/restchain.py]: I do it to selec the dataset in the origins list when there's more than one. Thus, grandparent is assigned the correct id.

File: bigmler/reify/restutils.py (hunk 3 above)
Comment (mmerce) [inline bigmler/reify/restutils.py]: ok

File: bigmler/reify/restchain.py (hunk 2 above)
Comment (jaor) [inline bigmler/reify/restchain.py]: okay! python tricked me on this one: i was convinced that grandparent
would be local to the for loop, but it isn't.

Comment (mmerce) [inline bigmler/reify/restchain.py]: let me declare grandparent before. I'll add a couple of tests anyways.

**Pull request merged**